app/controllers/api/v1/targets_controller.rb

```diff
@@ -6,7 +6,9 @@ def index
       end
 
       def create
-        @target = current_user.targets.create!(target_params)
+        @target = Targets::CreateService.new(current_user, target_params).call
+      rescue MaxAllowTargetsError
+        head :bad_request
```

Comment:
Return an error message please and a test for it

Reply:
And probably `:unprococessable_entity` fits better in this case
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422